pkg/graph/resolvers/operational_solution_subtask_test.go

```diff
@@ -65,3 +75,49 @@ func (suite *ResolverSuite) TestOperationalSolutionSubtaskDelete() {
 	suite.NoError(err)
 	suite.EqualValues(numDeletedRows, 1)
 }
+
+func (suite *ResolverSuite) TestOperationalSolutionSubtaskDataLoader() {
+	needType := models.OpNKManageCd
+	plan := suite.createModelPlan("Plan For OpSolS 1")
+	need, err := suite.testConfigs.Store.OperationalNeedGetByModelPlanIDAndType(suite.testConfigs.Logger, plan.ID, needType)
+	suite.NoError(err)
+	sol1, _ := OperationalSolutionInsertOrUpdateCustom(suite.testConfigs.Logger, need.ID, "AnotherSolution", nil, suite.testConfigs.Principal, suite.testConfigs.Store)
```

Comment:
NIT: Personal preference, I tend to prefer more verbose names like "solution1" as I believe it keeps the code more readable. It's a very minor and obviously optional thing.